test/service_binding_test/service_binding_test.go

```diff
@@ -675,6 +677,115 @@ var _ = DescribeTestsFor(TestCase{
 								})
 
 								if testCase.async {
+									When("maximum polling duration is reached while polling", func() {
+										var oldCtx *TestContext
```

Comment:
this is a buggy approach to do context switching because it swaps out the valid Basic auth credentials - i've tried to fix this bug in the transfer instance PR and the tests there. (Idea is that after doing the context swap and reverting to the old context, its basic credentials wont work